tools/docker/builder/openwrt/toolchains/entrypoint.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/sh
+
+cmd="$@"
+
+# Check if the command argument is a service command
+if [ "${cmd}" = "show-launch-script" ] ; then
+    cat /usr/local/bin/openwrt-env
+    exit
+fi
+
+FILE_SYSTEM_OUTPUT=/fs-output
+ROOT=/workspace
+WORK_DIR=${ROOT}
+
+# Create a user and his primary group that match a host's user inside a Docker
+# container.
+# Note: We don't create a user's home directory because it is already created by
+# mounting a host's user .ssh directory in 'docker run' command.
+groupadd -g ${GROUP_ID} ${GROUP_NAME}
+useradd -g ${GROUP_ID} -s "/bin/bash" -u ${USER_ID} ${USER_NAME}
```

Comment:
Please refer to `toolchains/rdkb/Dockerfile` - I did this differently there also without creating home directory since I map the current user home directory into the container.